paddle/fluid/operators/parallel_do_op.cc

```diff
@@ -166,8 +165,6 @@ class ParallelDoOp : public framework::OperatorBase {
 
       workers.emplace_back(
           framework::Async([program, cur_scope, place, block, place_idx] {
-            // Give the thread an id to distinguish parallel block with same id.
```

Comment:
Why remove these lines?

Reply:
I'm no longer using this solution to track threads.